integration-cli/docker_cli_inspect_test.go

```diff
@@ -353,13 +353,21 @@ func (s *DockerSuite) TestInspectByPrefix(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectStopWhenNotFound(c *check.C) {
-	runSleepingContainer(c, "--name=busybox", "-d")
-	runSleepingContainer(c, "--name=not-shown", "-d")
-	out, _, err := dockerCmdWithError("inspect", "--type=container", "--format='{{.Name}}'", "busybox", "missing", "not-shown")
+	runSleepingContainer(c, "--name=busybox1", "-d")
+	runSleepingContainer(c, "--name=busybox2", "-d")
+	out, _, err := dockerCmdWithError("inspect", "--type=container", "--format='{{.Name}}'", "busybox1", "busybox2", "missing")
+
+	c.Assert(err, checker.Not(check.IsNil))
+	c.Assert(out, checker.Contains, "busybox1")
+	c.Assert(out, checker.Contains, "busybox2")
+	c.Assert(out, checker.Contains, "Error: No such container: missing")
```

Comment:
I think this test can be improved by asserting that a specific stream (stdout vs stderr) containers this text, instead of the combined stream.

If you change `dockerCmdWithError()` to use the `dockerCmdWithResult()`, you can check `result.Stderr()`, and `result.Stdout()`